tools/ctl/localfiles/localfiles.go

```diff
@@ -11,20 +11,23 @@ import (
 	"github.com/pkg/errors"
 )
 
-// These are the different folders used for local storage.
-// 		./assets/		// AssetStore used to cache logs, artifacts, and other assets manipulated by ctl.
-//		./rundex/       // The local metadata about runs and attempts, normally generated by local smoketets benchmarks.
-
 const (
-	TempRoot = "/tmp/oss-rebuild/"
+	tempRoot = "/tmp/oss-rebuild/" // The directory where all the local files are stored
```

Comment:
again, this default should not be here. a better design would be to pass around a chroot with this abspath such that it's configured at as high a level as possible (i.e. injected at the program root). Having this be defined within a library is a code smell.

Reply:
I don't think I have super strong opinions on this, but I think I'd prefer to punt to a separate PR. That change would affect a few other usages of the localfiles and further affect the signatures in the library.

Reply:
yeah separate PR is fine